release/ray_release/scripts/ray_bisect.py

```diff
@@ -26,7 +26,7 @@ def main(test_name: str, passing_commit: str, failing_commit: str) -> None:
 
 def _bisect(test_name: str, commit_list: List[str]) -> str:
     test = _get_test(test_name)
-    while len(commit_list) > 1:
+    while len(commit_list) > 2:
```

Comment:
termination condition is 2 because the list always contains at least the passing and failing revisions